core/src/main/java/tc/oc/pgm/command/CancelCommand.java

```diff
@@ -18,11 +21,15 @@
       aliases = {"cancel", "cancelrestart", "cr"},
       desc = "Cancels all countdowns",
       perms = Permissions.STOP)
-  public void cancel(Audience audience, Match match) {
+  public void cancel(Audience audience, CommandSender sender, Match match) {
     if (RestartManager.isQueued()) {
       match.callEvent(new CancelRestartEvent());
       audience.sendMessage(
           TranslatableComponent.of("admin.cancelRestart.restartUnqueued", TextColor.RED));
+      ChatDispatcher.broadcastAdminChatMessage(
+          TranslatableComponent.of("admin.cancelRestart.broadcast", TextColor.GRAY)
+              .args(UsernameFormatUtils.formatStaffName(sender, match)),
+          match);
```

Comment:
I think this is over-verbosity on messages, we already have restart message.

Reply:
Just to clarify, are you referring to the `/queuerestart` broadcast as well, or should both be removed?